explore/cpp/utility.h

```diff
@@ -232,7 +232,7 @@ class NumberUtils
 class StringUtils
 {
 public:
-  static inline string to_string(size_t& n)
+  static inline string to_string(size_t n)
```

Comment:
Doesn't look like this is on the hot path at the moment so not a real issue. Just seemed odd to pass by value but I see this avoids an awkward new variable.